server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
```

Comment:
The naming convention for constants should be in uppercase letters, so 'PORT' and 'HOSTNAME' should be 'PORT' and 'HOSTNAME'.